src/common-lib/urls/urls.ts

```diff
@@ -594,6 +602,13 @@ export const OAK_PAGES: {
     configType: "internal",
     pageType: "lesson-overview",
   }),
+  "pupil-lesson": createOakPageConfig({
+    pathPattern:
+      "/pupils/beta/programmes/:programmeSlug/units/:unitSlug/lessons/:lessonSlug",
```

Comment:
I think you need to miss out the beta in this path so that it points to a valid url